exporters/otlp/test/otlp_http_metric_exporter_test.cc

```diff
@@ -196,13 +196,15 @@ class OtlpHttpMetricExporterTestPeer : public ::testing::Test
     auto client                = mock_otlp_client.second;
     auto exporter = GetExporter(std::unique_ptr<OtlpHttpClient>{mock_otlp_http_client});
 
+#  if 0
```

Comment:
Probably just remove the dead code?

Reply:
Given how metrics is still in development, not sure if this is:
- dead code (ok to remove then)
- not yet used code (which better stay)

so I took the safe choice.

Reply:
@owent 

Please advise on drop / keep dead code, this comes from:

commit 24d5100ada0c6cb9ccadb5827e4889e657ed8644
Author: WenTao Ou <owentou@tencent.com>
Date:   Tue Jul 19 15:59:24 2022 +0800

Reply:
> Given how metrics is still in development, not sure if this is:
> 
>     * dead code (ok to remove then)
> 
>     * not yet used code (which better stay)
> 
> 
> so I took the safe choice.

Yes, these codes can be removed. I forgot them and sorry for that.

Reply:
Thanks for confirming. Removed.